### PR TITLE
Service Fabric 8.0 CU2 Release Notes

### PR DESCRIPTION
**Pull request by sukanyamsft**

### CODE REVIEW

File: release_notes/Service_Fabric_ReleaseNotes_80CU1.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Microsoft Azure Service Fabric 8.0 Release Notes
...
... tries to verify the state of the known volumes in the metadata.db file but since the cluster is being upgraded the AzureFileDiskVolumeDriver services is not running. Each volume is given some time to respond before Docker moves on so with a sufficiently large number of volumes the rollback can be initiated before Docker is fully booted.<br>**Impact**: This impacts only customers using Docker Containers's via SF that have a large number of containers using the AzureFileDiskVolumeDriver.<br>**Change**: During cluster upgrade, the solution temporarily renames the metadata.db file such that Docker will boot without realizing that there were any previous volumes. This allows the cluster configuration to finalize and then Docker is restarted with the previous metadata file. When Docker boots the second time it has the same issue however, because the state of the upgrade is complete the AzureFileDiskVolumeDriver is started and immediately starts responding to Docker's queries so the cluster reaches the healthy state. If for any reason this causes harm (which is not expected), the Setup config entry "EnableDockerVolumeMetadataWorkaround" can be set to "false" to disable this workaround. |
```
Comment (peterpogorski) [inline release_notes/Service_Fabric_ReleaseNotes_80CU1.md:55]: Docker Containers's -> Docker container's

File: release_notes/Service_Fabric_ReleaseNotes_80CU2.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+|Service Fabric Runtime| Ubuntu 16 <br> Ubuntu 18 <br> Windows | NA <br>  NA <br> 8.0.521.9590  |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 8.0.521.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 5.0.521  <br> 8.0.521 <br> 8.0.521 <br> 8.0.521 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.0 |
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 8.0 Release Notes
```
Comment (peterpogorski) [inline release_notes/Service_Fabric_ReleaseNotes_80CU2.md:18]: Second refresh release

File: release_notes/Service_Fabric_ReleaseNotes_80CU1.md (hunk 1 above)
Comment (sukanyamsft) [reply release_notes/Service_Fabric_ReleaseNotes_80CU1.md:55]: Done

File: release_notes/Service_Fabric_ReleaseNotes_80CU2.md (hunk 2 above)
Comment (sukanyamsft) [reply release_notes/Service_Fabric_ReleaseNotes_80CU2.md:18]: Done

**Review by peterpogorski — APPROVED** [review]

**Pull request merged**